modules/data-entry/src/main/frontend/src/vocabQuery/InfoBox.jsx

```diff
@@ -0,0 +1,131 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React from "react";
+import classNames from "classnames";
+import { withStyles, Avatar, Button, Card, CardActions, CardContent, CardHeader, ClickAwayListener, Grow, IconButton, Link, Popper, Tooltip, Typography } from "@material-ui/core";
+import CloseIcon from '@material-ui/icons/Close';
+
+import QueryStyle from "./queryStyle.jsx";
+
+// Component that renders a dialog with term info for a single vocabulary term.
+//
+function InfoBox(props) {
+  const { open, infoboxRef, vocabulary, onClose, term, infoAboveBackground, browserOpened, openBrowser, onClickAway, classes } = props;
```

Comment:
Rename openBrowser to `onOpenBrowser` (use the `on<Event>` naming convention to clarify that property is a fuction).

Reorder props for better readability.
```suggestion
  const { open, infoboxRef, vocabulary, term, infoAboveBackground, browserOpened, onOpenBrowser, onClickAway, onClose, classes } = props;
```